From 33dc04e9de44aa7a1f72544ce00f2fe239c3a62c Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Fri, 2 Feb 2018 03:30:53 +0000
Subject: [PATCH] fix build with glibc 2.27

With glibc 2.27 memfd_create() is behind a _GNU_SOURCE guard, so use
AC_USE_SYSTEM_EXTENSIONS to define it.

(From OE-Core rev: 88b3d730021107985ea749c92e52a323690f87dc)

Ref: http://git.yoctoproject.org/cgit.cgi/poky/commit/?id=fc56788a3201daf1b1a9f3b96d5501ffe489dcd1
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 0c98875..ebf4dee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -28,6 +28,8 @@ AC_INIT([libxshmfence], [1.2],
 AC_CONFIG_SRCDIR([Makefile.am])
 AC_CONFIG_HEADERS([config.h])
 
+AC_USE_SYSTEM_EXTENSIONS
+
 # Initialize Automake
 AM_INIT_AUTOMAKE([foreign dist-bzip2])
 
-- 
2.14.1

